docs/GettingStartedGuide.md

```diff
@@ -56,4 +62,14 @@ More details how to use the CatalogClient are placed in the [Catalog usage examp
 
 ## <a name="examples"></a> Layer Usage Example
 
-[Versioned Layer Read example](./examples/nodejs-read-versioned-layer.md) shows how to use VersionLayerClient and how to retrieve data, metadata and partition data from versioned layer, using OLP SDK for TypeScript.
\ No newline at end of file
+[Versioned Layer Read example](./examples/nodejs-read-versioned-layer.md) shows how to use VersionLayerClient and how to retrieve data, metadata and partition data from versioned layer, using OLP SDK for TypeScript.
+
+## <a name="license"></a> License
```

Comment:
Do we need additional licence information here?
Should be in the example source code itself.

Reply:
Removed